test_files/test.enc.yaml

```diff
@@ -0,0 +1,38 @@
+enc_var: ENC[AES256_GCM,data:JhiYyf1r8D33,iv:GZ9Mvc/XlWP2tJZ/Sv28i8PfgIZG6ytbv/tgnAds1mM=,tag:ek76wqWzcXkoPBS/TW04ag==,type:str]
```

Comment:
Hmm, I'm getting `panic: qa: cannot decrypt file: Error getting data key: 0 successful groups required, got 0` for this, should we commit a dummy private key to this repo?

Reply:
solved with `gpg --import ./test_files/sops_functional_tests_key.asc`